### ISSUE TITLE
Migration timestamp in database should be the execution timestamp, not the creation timestamp

### ISSUE THREAD

**Issue by MaikeMota** [typeorm/typeorm#6827]
**Issue type:**
**[x] bug report**
[ ] feature request
[ ] documentation issue

**Database system/driver:**

[ ] `cordova`
[ ] `mongodb`
[ ] `mssql`
[ ] `mysql` / `mariadb`
[ ] `oracle`
**[x] `postgres`**
[ ] `cockroachdb`
[ ] `sqlite`
[ ] `sqljs`
[ ] `react-native`
[ ] `expo`

**TypeORM version:**

[ ] `latest`
[ ] `@next`
**[x] `0.2.25`**

When running a migration, the timestamp in the migration table in the database is inserting the same timestamp as the creation of the file ( the one prepending at the filename).
![image](https://user-images.githubusercontent.com/4526602/94895600-2771af00-0462-11eb-931c-9e93aa92fb8c.png)

**Steps to reproduce or a small repository showing the problem:**

Run any migration.

**Expected behavior**
When a migration is created, a timestamp is prepended at his filename, this is a good way for preventing filename collision, but this is not the timestamp when the migration is really applied in the database, that still need to be developed and tested, maybe run in different environments before really executes in a production database. 

What I expected to see in the timestamp column in the database is exactly when that migration has executed in that database. If I need to track down a bug or some misbehavior, is easier to see if the that migration could be the cause or contribute for that.

**Comment by imnotjames** [comment]
The timestamp in the column (and file/class) is for determining which migrations have run, which still need to be run, and the order.

Adding an "executed at" column to it (amusingly) would be pretty difficult to do, but it sounds like that's what you're requesting?

**Comment by MaikeMota** [comment]
Hm... 
It's look like I had a misunderstanding about the timestamp column in TypeORM migration table...

But, from my previously experience with others ORM, and another languages, the timestamp values from migration collection/table always is the execution date, for historical purpose.

Adding the "executed at" column will provide the information that will be utile to know when the migrations has been executed on the database. 

Should I change this issue to a Feature Request?

**Comment by kendallroth** [comment]
I agree that having an `executed_at` timestamp could be quite useful (only for historical purposes, not used for anything)

**Comment by Thomas-1985** [comment]
This would be very useful for many projects. +1 :)

**Comment by peterbherman** [comment]
> If you believe this issue is still relevant, feel free to comment below and the maintainers may reopen it. Thank you for your contribution.

I think this issue is still relevant.

**Comment by cgrafton** [comment]
> > If you believe this issue is still relevant, feel free to comment below and the maintainers may reopen it. Thank you for your contribution.
> 
> I think this issue is still relevant.

I also think this issue is still relevant
